docs/src/rsh/step/index.md

```diff
@@ -464,6 +464,22 @@ closeTo( Who, after, nonNetPayAmt )
 ```
 
 @{ref("rsh", "closeTo")} Has participant `{!rsh} Anybody` make a publication, then transfer the `{!rsh} balance()` and the non-network pay amount to `{!rsh} Who` and end the DApp after executing the function `{!rsh} after` in a step.
+
+As an example, codifying the prose above would look like:
+
+```reach
+// Has participant Who make a publication
+  Who.publish();
```

Comment:
`Anybody`, not `Who`

Reply:
Also, I don't like that the comments don't line up with the code

Reply:
In this example, we're talking about this participant, `Who`. "Who" is publishing and "Who" is receiving a payment transfer. Would it be confusing to refer to "Anybody?"
Perhaps it's better to wrap "Who" in monotype in the comment?
// Has participant `Who` make a publication

Reply:
My point is that `Who`, this participant, is *not* the one publishing. `Anybody` is publishing. (I.e. your code sample is wrong)